src/evo/netinfo.cpp

```diff
@@ -58,3 +58,11 @@ NetInfoStatus MnNetInfo::AddEntry(const std::string& input)
     }
     return NetInfoStatus::BadInput;
 }
+
+std::string MnNetInfo::ToString() const
+{
+    // Extra padding to account for padding done by the calling function.
+    return strprintf("MnNetInfo()\n"
```

Comment:
let's do it in one line? \n are not friendly for logs without a good reason; especially if that's part of complex object such as SimplifiedMNListEntry's logging

`return strprintf("MnNetInfo(m_addr=CService(addr=%s, port=%u))"`

Reply:
Not feasible, see `ToString()` for `ExtNetInfo` (see below, WIP code). Would like `MnNetInfo` to retain a similar structure.

https://github.com/dashpay/dash/blob/17f34c6bc7c1fa9f326353e7057153ddf2b32f94/src/evo/netinfo.cpp#L320-L327

When purpose codes are introduced, the structure gets more complicated (see below, WIP code)

https://github.com/dashpay/dash/blob/7a9bfa2873fcb53b9f5f61b709f95adbc4c5f666/src/evo/netinfo.cpp#L577-L591

Which `MnNetInfo` will be emulating (see below, WIP code)

https://github.com/dashpay/dash/blob/7a9bfa2873fcb53b9f5f61b709f95adbc4c5f666/src/evo/netinfo.cpp#L336-L342